### DISCUSSION TITLE
Step: Infinite loop detected

### DISCUSSION THREAD

**Discussion by s1lv3rdr4g0ngames** [ramokz/phantom-camera#409] [Question / Help Wanted] [answered]
So having done a lot of testing, this appears to be uniqely related to my player controller, but I'm wondering if anyone has had a similar issue and if so how to fix it.

My player controller is a bit of a hack, it's a node with two character bodies (each with a camera). One walks the other flies, and the root node manages which is active. 

When I try to load another scene, I get the error `step: Infinite loop detected. Check set_loops() description for more info.` and I cannot for the life of me figure out why. My guess is that it's on exit_tree and it's breaking the tween to next active camera, but I cannot find how to stop that. In theory I could just ignore it since it happens on load and is caught by an error but it bothers me. 

Any suggestions?

EDIT: So I found a symptom. It''s the moment between ready and the first process frame. Doing this silliness will stifle it with no knock on effects. Though I don't exactly know why this is happening at all. It doesn't happen in transition between example scenes, or between an example scene and my scene. Just from my scene to any other scene. Maybe something is getting improperly freed on the scene leavning end, but I can't find what. 
![image](https://github.com/user-attachments/assets/b24065fa-e6f6-4d24-af27-2416b360b7b7)

**Comment by ramokz** [comment, marked as answer]
Would you be able to provide a minimal project that reproduces the issue?
Otherwise, it's borderline impossible to give any advice or help when it's not a simple reproducible issue.

**Reply by s1lv3rdr4g0ngames** [reply]
Sigh I'm a moron. The ONLY actual other tween in my entire project is a shader for speedlines. And it calls set_loops() with no parameters.
